src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -975,6 +982,33 @@ public class ShowHideConsoleMenuItemConverter : IValueConverter
         #endregion
     }
 
+    public class ConsoleHeightToBooleanConverter : IValueConverter
```

Comment:
A new converter to convert console height to show console state: console height 0 -> false, console height > 0 -> true